keps/sig-api-machinery/3903-unknown-version-interoperability-proxy/README.md

```diff
@@ -254,14 +254,24 @@ the number if needed, but upgrades aren't that frequent and few resources are
 changed on releases, so these requests should not be common. We will count them
 with a metric.
 
+There could be a large volume of requests for a specific resource which might result in the identified apiserver to be unable to serve the proxied requests. Although this scenario should not occur too frequently, since resource types that have large request traffic should not undergo upgrades too often.
```

Comment:
```suggestion
There could be a large volume of requests for a specific resource which might result in the identified apiserver being unable to serve the proxied requests. This scenario should not occur too frequently, since resource types which have large request volume should not be added or removed during an upgrade -- that would cause other problems, too.
```